version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "0.6.1-SNAPSHOT"
\ No newline at end of file
+version in ThisBuild := "0.6.1-1-SNAPSHOT"
```

Comment:
Not sure about this change. Definitely we should not change that on master (and in upstream PR). It will be helpful to change version on our branch, but maybe removing SNAPSHOT and publishing it as our release (as we do currently). @sapkos what do you think?